lib/Backend/arm64/LowerMD.cpp

```diff
@@ -7067,7 +7067,7 @@ LowererMD::GenerateFastInlineBuiltInMathFloorCeilRound(IR::Instr* instr)
 
     // MSR FPSR, xzr
     IR::Instr* setFPSRInstr = IR::Instr::New(Js::OpCode::MSR_FPSR, instr->m_func);
-    setFPSRInstr->SetSrc1(IR::RegOpnd::New(nullptr, RegZR, TyMachReg, instr->m_func));
```

Comment:
>setFPSRInstr [](start = 4, length = 12)

Legalizer should be able to do such things but since assert in encoder will catch future issues its ok for now.